README.md

```diff
@@ -1,131 +1,231 @@
+* "Counting the numbers of threads of a process"
+* "Changing the current databse connection count for a service"
+* "Swinging traffic from one to another"
+
+should be possible with `ae`.
+
+There are more "practical" and "impact-oriented" tasks, and these probably need extra functionality, which they add to the lightweight `aer` tool.
+
+<!-- QUICKSTART -->
+<h2 id="quickstart">Quickstart</h2>
+
+> This section is reserved for future installation instructions as well as an example of integration.
+
+<!-- USAGE -->
+<h2 id="usage">Usage</h2>
+
+There are a number of commands for `ae`.
+
+These are shown here in _alphabetical_ order.
+
+<details>
+  <summary><code>allocate</code></summary>
+   
+  &nbsp;
+  
+  Resources are reserved and prerequisites can be managed, but it **does not** start. It will not work if the resources are not available.
+    
+  ```
+  ae allocate
+  ae allocate cell
+  ae allocate pod
+  ```
+  
+</details>
+
+<details>
+  <summary><code>check</code></summary>
+  
+  &nbsp;
+  
+  Checks the nodes of the cluster and returns the current serving status with the given list of services.
+    
+  ```
+  ae check <cidr <cidr> | ip <ip>> <service, ...>
+  ```
+  
+</details>
+
+<details>
+  <summary><code>discover</code></summary>
+  
+  &nbsp;
+  
+  Scans the complete network or cluster of nodes and returns information about it, including the version.
+    
+  ```
+  ae discover <cidr <cidr> | ip <ip>>
+  ```
+    
+</details>
+
+<details>
+  <summary><code>free</code></summary>
+  
+  &nbsp;
+  
+  It frees the resources and destroys the prerequisites that were started. It will fail if the resources cannot be freed or do not exist.
+    
+  ```
+  ae free
+  ae free cell
+  ae free pod
+  ```
+    
+</details>
+
+<details>
+  <summary><code>logs</code></summary>
+  
+  &nbsp;
+  
+  This option will accept aruguments and return or save some kind of logs.
+    
+  ```
+  ae logs <options>
+  ```
+    
+</details>
+
+<details>
+  <summary><code>oci</code></summary>
+  
+  &nbsp;
+  
+  Here the [OCI CLI interface](https://github.com/opencontainers/runtime-tools/blob/master/docs/command-line-interface.md) is implemented with the respective subcommands.
+    
+  ```
+  ae oci
+  ae oci create
+  ae oci delete
+  ae oci kill
+  ae oci start
+  ae oci status
+  ```
+    
+</details>
+
+<details>
+  <summary><code>start</code></summary>
+  
+  &nbsp;
+  
+  It will run the rescource directly.
+    
+  ```
+  ae start
+  ae start executable, exe
+  ae start container # Note this has an alias: 'ae oci start'
+  ```
+    
+</details>
+
+<details>
+  <summary><code>stop</code></summary>
+  
+  &nbsp;
+  
+  It will stop the rescource directly.
+    
+  ```
+  ae stop
+  ae stop executable, exe
+  ae stop container # Note this has an alias: 'ae oci kill'
+  ```
+    
+</details>
+
+<!-- PHILOSOPHY -->
+<h2 id="philosophy">Philosophy</h2>
+
+**This project has a few philosophical principles.**
+    
+### **Less is more**
```

Comment:
i don't think we need additional formatting inside the existing header styles please.

Reply:
5b43995 fixed it. more compact, thank you!